sqlx-data.json

```diff
@@ -1054,15 +1086,16 @@
       ]
     }
   },
-  "c91b34f1d6c7fd42d0aa51e9b75bbb15c6bf04ffeb4fdece62470bc57ed31761": {
-    "query": "INSERT INTO owners (owner_id, mod_name, checksums) VALUES ($1, $2, $3)",
+  "f2c8d6ca44a490a68b1d13b088d5171bd87ddcbf0f933eaaa2e1f09fad20a1bb": {
+    "query": "INSERT INTO owners (owner_id, mod_name, checksums, is_team) VALUES ($1, $2, $3, $4)",
     "describe": {
       "columns": [],
       "parameters": {
         "Left": [
           "Int8",
           "Text",
-          "VarcharArray"
+          "VarcharArray",
+          "Bool"
```

Comment:
This file is generated via `cargo sqlx prepare`, you don't need to fill it manually.